classes/submission/sectionEditor/SectionEditorSubmissionDAO.inc.php

```diff
@@ -1305,6 +1305,61 @@ function getProofreaderStatistics($journalId, $proofreaderId = null) {
 
 		return $statistics;
 	}
+	
+	/* Draft editorial https://github.com/ubiquitypress/OJS-Draft-Editorial/commit/9135893848e9d40923c105a2cafc59b0529659c9 */
+	function recordDraftDecision($articleID, $Key, $seniorEditorID, $juniorEditorID, $Subject, $Body, $Note, $Decision, $Status) {
```

Comment:
We use `$lowerCamelCase` conventions for PHP variables. For consistency, `$articleID` should be `$articleId`, `$Key` should be `$key`, `$Subject` should be `$subject`, etc.